spec/mongoid/traversable_spec.rb

```diff
@@ -251,4 +251,108 @@
       end
     end
   end
+
+  describe "#discriminator_key" do
+
+    context "when the discriminator key is not changed" do 
+
+      it "equals _type" do
+        expect(Instrument.discriminator_key).to eq("_type")
+      end
+
+      it "the global discriminator key is _type" do
+        expect(Mongoid.discriminator_key).to eq("_type")
+      end
+
+      it "child discriminator keys equal _type" do
+        expect(Piano.discriminator_key).to eq("_type")
+        expect(Guitar.discriminator_key).to eq("_type")
+      end
+    end
+    
+    context "when the discriminator key is changed at the global level" do 
+
+      before do
+        Mongoid.discriminator_key = "hello"
+      end
+
+      after do
+        Mongoid.discriminator_key = "_type"
+      end
+
+      it "the value changes" do 
+        expect(Mongoid.discriminator_key).to eq("hello")
+      end
+
+      it "all discriminator keys change" do 
```

Comment:
This should probably be 3 separate tests with each test having its own descriptive label as to what the test is testing.